libbeat/logp/logp.go

```diff
@@ -21,8 +24,18 @@ type Logging struct {
 	ToSyslog  *bool `config:"to_syslog"`
 	ToFiles   *bool `config:"to_files"`
 	Level     string
+	Metrics   LoggingMetricsConfig `config:"metrics"`
```

Comment:
I tried initially but didn't work out due to circular imports (cfgfile imports logp).